lib/msf/core/post/windows/services.rb

```diff
@@ -655,6 +663,94 @@ def meterpreter_service_list
 
           services
         end
+
+        #
+        # Start a service using sc.exe.
+        #
+        # @param name [String] Service name (not display name)
+        # @param server [String,nil] A hostname or IP address. Default is the
+        #   remote localhost.
+        #
+        # @return [Integer] 0 if service started successfully, 1 if it failed
+        #   because the service is already running, 2 if it is disabled
+        #
+        # @raise [RuntimeError] starting service failed
+        #
+        def _shell_service_start(service_name, server = nil)
+          host = server ? "\\\\#{server}" : nil
+          timeout = 62 # sc.exe default RPC connection timeout is 60 seconds
+
+          fingerprint = Rex::Text.rand_text_alphanumeric(6..8)
+
+          res = cmd_exec("sc #{host} start #{service_name} && echo #{fingerprint}", nil, timeout).to_s
```

Comment:
```suggestion
          res = cmd_exec("sc #{host} start #{service_name} && echo #{fingerprint}", time_out: timeout).to_s
```

Reply:
Are you sure this works? I'm not sure about Windows but I just tested on Linux and it certainly does not.

Reply:
I will not implement this change. This change is not necessary at best and broken at worst.